twpca/model.py

```diff
@@ -415,6 +423,14 @@ def predict(self, X=None):
 
             return pred
 
+    def dump(self):
+        """Serializes model variables"""
+        return {k : self._sess.run(v) for k, v in self._vars.items()}
+
+    def load(self, new_vars):
+        """Assigns model variables from numpy arrays"""
+        [self._sess.run([tf.assign(self._vars[k], v)]) for k, v in new_vars.items()]
```

Comment:
only call `self._sess.run` once. you could do this by jus having the list comprehension be the argument to `run`.